tests/utils/index.js

```diff
@@ -1260,6 +1260,8 @@ const logFeedbackDocs = async (test) => {
 
 const isMinimumChromeVersion = process.env.CHROME_VERSION === MINIMUM_BROWSER_VERSION;
 
+const escapeBranchName = (branch) => branch?.replace(/[/|_]/g, '-');
```

Comment:
question: I'm curious as to why we maintain a replica of the implementation in the tests folder and not use the actual implementation.

Reply:
I considered using the same implementation everywhere, but decided against it, because if the test implementation just calls the same function that the code calls, then you're not really testing anything. It's like checking that if you call a function twice it returns the same thing.

It's sort of a moot point here because we can plainly see the implementation is the same, but it's more of a philosophical decision to keep the implementation and test code separate.